imdb/views.py

```diff
@@ -0,0 +1,56 @@
+import json
+import pdb
+import sys
+import traceback
+
+import requests
+
+import SearchResults
+
+
+from bs4 import BeautifulSoup
+from django.http import HttpResponse
+from django.shortcuts import render
+from django.core.paginator import Paginator, PageNotAnInteger, EmptyPage
+
+
+
+def search_movie(request):
+
+    movie = request.GET.get('q','')
+    error, status, movie_list = SearchResults.get_search_movies(movie)
+
+    if error:
+        data = json.dumps(movie_list)
```

Comment:
json.dumps is used to convert the data we have in the movie_list in the particular json format . so that when we display the given data on the page it is displayed in the json format further using the Paginator lmodule